packages/dds/tree/.eslintrc.cjs

```diff
@@ -25,8 +25,12 @@ module.exports = {
 				caughtErrorsIgnorePattern: "^_",
 			},
 		],
-		// This rule can be removed once the client release group has eslint-config-fluid 5.2.0+
+
+		// TODO: Remove this override once dependency on eslint-config-fluid has been updated to 5.2.0+
 		"import/order": "off",
+
+		// TODO: Remove these overrides once this config has been updated to extend the "strict" base config.
+		"@typescript-eslint/explicit-member-accessibility": "error",
```

Comment:
Pre-updating rule pattern here so we can start enforcing rules 1 at a time until we're close enough to just upgrade the base we extend.